ray-curator/ray_curator/stages/filters/__init__.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Since these are all text filters I believe they should live under `stages/filters/text`

Reply:
I believe there are plans to try to support using them on captions, etc.? Not sure what that will look like yet though.

Reply:
I think we can skip all organisational conversations for now ? We will have to do refactors based on Product/Marketing anyways later